edit/edit.js

```diff
@@ -508,7 +517,7 @@ window.onbeforeunload = () => {
   if (isCleanGlobal()) {
     return;
   }
-  updateLintReport(null, 0);
+  updateLintReportIfEnabled(null, 0);
   return confirm(t('styleChangesNotSaved'));
```

Comment:
Also, at least for me, `useHistoryBack` is always `false`.. so the code in `goBackToManage` function isn't ever called.